### DISCUSSION TITLE
How can I turn off verbose prisma client logging to stdout?

### DISCUSSION THREAD

**Discussion by jde** [prisma/prisma#24372] [Q&A] [answered]
### Question

How can I turn off verbose prisma client logging to stdout?

Each time I run a query, I'm getting a huge amout of log output, such as 

```
prisma:client prisma.queuedMessage.findMany({
  where: {
    tripId: "xyz",
    type: "onboarding"
  }
}) +0ms
prisma:client Generated request: +0ms
prisma:client {
  "modelName": "QueuedMessage",
  "action": "findMany",
  "query": {
    "arguments": {
      "where": {
        "tripId": "xyz",
        "type": "onboarding"
      }
    },
    "selection": {
      "$composites": true,
      "$scalars": true
    }
  }
}
 +0ms
```

I have tried passing log options into my `new PrismaClient()` statement, but it does not effect this level of logging. For example, I am playing with:

```
const prisma =
  global.prisma ||
  new PrismaClient({
    log: [
      {
        emit: 'event',
        level: 'query',
      },
      {
        emit: 'stdout',
        level: 'error',
      },
      {
        emit: 'stdout',
        level: 'info',
      },
      {
        emit: 'stdout',
        level: 'warn',
      },
    ],
  })

prisma.$on('query', e => {
  console.log('Query: ' + e.query)
  console.log('Params: ' + e.params)
  console.log('Duration: ' + e.duration + 'ms')
})

```

When I do so I get the expected event output _in addition_ to the unwanted verbose output. I've played with many other values in the invocation of the client and they work as expected. It's just that I get all that output on top of what I'm asking for.

How can I clean up my stdout?

### How to reproduce (optional)

I am using prisma with the MySQL connecter in a Next.js app.

### Expected behavior (optional)

Prisma client only logs what is requested.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment, marked as answer]
By default Prisma Client is pretty silent. Do you have `DEBUG=*` environment variable set?

**Reply by jde** [reply]
Yes. That was it. Thanks so much for the help!

**Comment by jde** [comment]
I found it. The logs in question are _debug_ and not _logging_. I am able to control them via the DEBUG env var.
